maintenance/importConfig.ts

```diff
@@ -109,6 +110,8 @@ const processedDir = path.join(
 	"config/devices",
 );
 
+const paramsRegex = /\[0x([0-9]+)\]/;
```

Comment:
The params are HEX, so you'll need to include `a-f` in the character class too.

Reply:
And probably A-F too just to be sure, see https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/src/Devices.ts#L250

Reply:
I have added the `i` flag to regex so it's case unsensitive